src/main/java/com/bootcamp/demo/model/User.java

```diff
@@ -0,0 +1,84 @@
+package com.bootcamp.demo.model;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public final class User {
+    private final UUID userId;
+    private String email;
+    private String firstName;
+    private String lastName;
+
+    public User(String email, String firstName, String lastName) {
+        this.userId = UUID.randomUUID();
```

Comment:
Same here as for booking. Should the type for `userId` be String to avoid the issue that @mariatirlea pointed out on slack?